ansible/roles/jenkins-worker/files/docker-node-exec.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+## This script is designed to be enabled in /etc/sudoers for the `iojs` user,
+## the only privileged access that user has to Docker.
+## Since there is considerable access given by selecting arbitrary images and
+## execution commands, there are still security concerns and additions of new
+## images and changes to existing ones as well as the Bash that's executed
+## inside them should be monitored for malicious activity.
+
+set -e
+
+OPTIND=1
+image_base="rvagg/node-ci-containers"
```

Comment:
We should probably create a user for the Node.js project and use that instead of rvagg right?

Reply:
I see your comment elsewhere about the path not being important so seems like you have already acknowledged we should do something on that front.